api/src/test/java/uk/gov/dhsc/htbhf/claimant/entitlement/CycleEntitlementCalculatorIntegrationTests.java

```diff
@@ -0,0 +1,107 @@
+package uk.gov.dhsc.htbhf.claimant.entitlement;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import uk.gov.dhsc.htbhf.claimant.entity.Claimant;
+
+import java.time.LocalDate;
+import java.util.List;
+
+import static java.util.Arrays.asList;
+import static java.util.Collections.singletonList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.dhsc.htbhf.claimant.testsupport.ClaimantTestDataFactory.aValidClaimantBuilder;
+
+@SpringBootTest
+class CycleEntitlementCalculatorIntegrationTests {
+
+    @Autowired
+    private CycleEntitlementCalculator cycleEntitlementCalculator;
+
+    @DisplayName("Calculates the correct entitlement when there is a child under one,"
+             + " a child under four and neither child changes age during the payment cycle.")
+    @Test
+    void shouldCalculateCorrectEntitlementForNoChanges() {
+        Claimant claimant = aValidClaimantBuilder().build();
+        List<LocalDate> childrenDatesOfBirth = asList(LocalDate.now().minusMonths(6), LocalDate.now().minusYears(3));
```

Comment:
I would prefer to see these two dates created as their own variables with good names so its clearer.

Reply:
Done.